plenum/common/messages/fields.py

```diff
@@ -670,7 +670,8 @@ class ProtocolVersionField(FieldBase):
     _base_types = (int, type(None))
 
     def _specific_validation(self, val):
-        if val is None:
-            return
         if not PlenumProtocolVersion.has_value(val):
             return 'Unknown protocol version value {}'.format(val)
+        if val != CURRENT_PROTOCOL_VERSION:
```

Comment:
Our goal it to make clients aware that they need to update if they don't send correct PROTOCOL_VERSION.
There are two problems here:
1) If there is no protocol version at all, then the base validation will be triggerred (since PROTOCOL VERSION is not optional anymore)
2) If there is unsupported protocol version, then thos method will be triggered, so we need to extend the error message with a request to update client code.